pkg/storageops/aws/aws.go

```diff
@@ -216,24 +176,31 @@ func (s *ec2Ops) waitAttachmentStatus(
 	return outVol, nil
 }
 
+func (s *ec2Ops) Name() string { return "aws" }
+
 func (s *ec2Ops) ApplyTags(
-	v *ec2.Volume,
+	v interface{},
 	labels map[string]string,
 ) error {
+
+	vol := v.(*ec2.Volume)
```

Comment:
vol, ok := v.(*ec2.Volume)
check if type cast is successful and avoid go panics.

Reply:
Fixed in incremental